colossalai/accelerator/npu_accelerator.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+from typing import Union
+
+import torch
+
+from .base_accelerator import BaseAccelerator
+
+try:
+    pass
```

Comment:
Is this removed by pre-commit? Use `import torch_npu # noqa`

Reply:
ok

Reply:
Done.